### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by MMGGYY66**
## Add some validations in the HTML form.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -275,32 +275,32 @@ <h3 id="contact2">
...
 
           <label for="last_name">Last Name</label>
-          <input type="text" name="last_name" placeholder="Last Name" maxlength="30" />
+          <input type="text" name="last_name" placeholder="Last Name" maxlength="30" required/>
         </div>
         
-        <label for="user_email">Email</label>
-        <input type="email" name="user_email" placeholder="Email Adress" id="contact-me-email" required />
+        <label for="email">Email</label>
+        <input type="email" name="email"  placeholder="Email Adress" pattern=".+@yahoo\.com"  id="email" value="default@example.com" required />
```
Comment (anagudelogu) [inline index.html:293]: - [ ] **[OPTIONAL]:** You did a good job working on this milestone! 🚀 However I suggest that you remove the pattern and the value attributes from your email field, you're limiting your app to only receive `yahoo.com` emails and you're also prepopulating the field with data which could lead to poor user experience. I strongly suggest you fix this! 🤓 🛠️

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
### Hi @MMGGYY66, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -275,32 +275,32 @@ <h3 id="contact2">
...
         to chat please get in touch.
       </h3>
-      <form action="https://formspree.io/f/mayvdjkb" method="POST" class="form apply-flex" id="contact-form">
+      <form action="https://formspree.io/f/xvoyekjg" method="post" class="form apply-flex" id="contact-form">
         
         <!-- Adding labels to all form elements for accessibility reasons and make them invisible by adding css rules display: none for the sake of Figma design" -->
-        <label for="user_fullname">Full Name</label>
-        <input type="text" name="user_fullname" placeholder="Full name" maxlength="30" />
+        <label for="name">Full Name</label>
+        <input type="text" name="name" id="name" placeholder="Full name" maxlength="30" required/>
```
Comment (anagudelogu) [inline index.html:282]: - [ ] There's a problem with the submission of your contact form. You have different input fields for the name on both the desktop and mobile versions hiding them depending on the type of device and marking them as required. This is causing your form to have unreachable fields preventing the correct submission.

Please consider refactoring your form, you only need one input field for the name (no need to separate it into First and Last names), and make sure that it is working as expected. 🤓 🛠️

**Review by anagudelogu — COMMENTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @MMGGYY66, 
I'm Andres.

⚠️ **NOTE:** YOUR PARTNER SUBMITTED **YOUR REPO** FOR REVIEW AND GOT MISTAKENLY APPROVED. PLEASE REMEMBER TO READ CAREFULLY ALL THE PROJECT REQUIREMENTS (INCLUDING THE WORK AND SUBMISSION MODE) BEFORE YOU START WORKING ON THEM, THIS MILESTONE SHOULD BE IMPLEMENTED IN BOTH PROJECTS. ⚠️ 

image


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**